### PR TITLE
[GH-771] WGS updates

### PR DESCRIPTION
**Pull request by jack-r-warren**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-771

### Changes

- Bump the WDL to 1.1.1 per Jessica Way
- Remove the fingerprinting since that is a per-sample thing (also per Jessica Way)
- Roll in Rex's v52 changes from GH-994 (#157)

### Review Instructions

- Is GH-994 still blocked? Are we not wanting this to be in because of where Cromwell v52 is and isn't?

### CODE REVIEW

**Review by rexwangcc — COMMENTED** [review]
Looks good, I'm not sure if this will work for Cromwell v52+ since https://github.com/broadinstitute/wfl/pull/157/files are not in yet.

File: api/src/wfl/module/wgs.clj [hunk 1]
```diff
@@ -106,7 +106,7 @@
                    (assoc input-key in-gs)
                    (assoc :destination_cloud_path (str out-gs out-dir))
                    (assoc :references (make-references ref-prefix))
-                   #_(merge fingerprinting) ;; Uncomment to enable fingerprinting
+                   (merge fingerprinting) ;; Uncomment to enable fingerprinting
```
Comment (ehigham) [inline api/src/wfl/module/wgs.clj:109]: remove the comment too!

Comment (jack-r-warren) [reply api/src/wfl/module/wgs.clj]: Oops, my bad. Guess this is staying commented anyway for right now? I'm still trying to sort out this ticket, I reverted the commit that uncommented this

**Comment by jack-r-warren** [comment]
Just went and pulled Rex's Cromwell v52 changes from GH-994 (#157) into this PR so this should work properly

**Review by rexwangcc — APPROVED** [review]
LGTM. (I moved GH-994 to the current sprint, we can close it together with this one once the PR gets in)

**Pull request merged**